src/_guides/asyncapi/what-is-asyncapi.md

```diff
@@ -101,14 +101,14 @@ AsyncAPI has gained more and more popularity since its version 2.0. Great tools
 ### Mocking & Testing
 
 - [Microrocks](https://microcks.io/)
-- [Virtualan](https://virtualan.io/)
+- [Beeceptor](https://beeceptor.com/)
```

Comment:
@ankitjaininfo Thanks for pointing this out!

However, does Beeceptor support AsyncAPI, since this article is about that specification? I couldn't find reference to it in the Beeceptor site and docs.